packages/calcite-components/src/utils/dom.ts

```diff
@@ -695,19 +727,29 @@ export async function whenAnimationDone(targetEl: HTMLElement, animationName: st
     targetEl.addEventListener(endEvent, onEndOrCancel);
     targetEl.addEventListener(cancelEvent, onEndOrCancel);
 
-    function onStart(event: AnimationEvent): void {
-      if (event.animationName === animationName && event.target === targetEl) {
-        clearTimeout(fallbackTimeoutId);
+    function onStart(event: TransitionOrAnimationEvent): void {
+      console.log("start");
```

Comment:
Moar console logs 🪵

Reply:
Yikes, sorry about that. All cleaned up now!